### ISSUE TITLE
I meet the zero gradient descent

### ISSUE THREAD

**Issue by Sniper970119** [huggingface/transformers#8493]
I want use transformers to do  text classification, I want code myself rather than use `TFBertForSequenceClassification`,so I write the model with `TFBertModel` and `tf.keras.laters.Dense`,but this is no gradient descent in my code, I try to find what wrong with my code but I can't. So I submit this issues to ask for some help.
my code is here:

Model:
![图片](https://user-images.githubusercontent.com/30463691/98935026-e15a4180-251d-11eb-87a5-0d23bcb7cf4a.png)

![图片](https://user-images.githubusercontent.com/30463691/98935038-e4edc880-251d-11eb-9c3d-ee32df987590.png)

and I know train data is test data,just for quick debug.
and when I train this model , 
![图片](https://user-images.githubusercontent.com/30463691/98934626-5ed18200-251d-11eb-9eb3-c0330a6398bf.png)

**Comment by thomwolf** [comment]
Hi @Sniper970119 do you mind posting this on the forum rather? It's here: https://discuss.huggingface.co

We are trying to focus the issues on bug reports and features/model requests.

Thanks a lot.

**Comment by Sniper970119** [comment]
> 
> 
> Hi @Sniper970119 do you mind posting this on the forum rather? It's here: https://discuss.huggingface.co
> 
> We are trying to focus the issues on bug reports and features/model requests.
> 
> Thanks a lot.

ok,I just post it on the forum.Thank for your reply.
